hack/traffic_flow_tests.sh

```diff
@@ -28,3 +28,5 @@ temp_file=$(mktemp)
 envsubst < ../hack/cluster-configs/ocp-tft-config.yaml > $temp_file
 
 ./tft.py $temp_file
+
+./print_results.py ft-logs/*RESULTS
```

Comment:
the `*RESULTS` file is deprecated. See [here](https://github.com/ovn-kubernetes/kubernetes-traffic-flow-tests/commit/1d03e96142a1b24ac1bbd728ebad140251ace641). Use `ft-logs/*.json`.

Maybe even:
```bash
OUTPUT_BASE="./ft-logs/result-$(date '+%Y%m%d-%H%M%S.%s')"

./tft.py --output-base "$OUTPUT_BASE" "$temp_file"

./print_results.py "$OUTPUT_BASE"*.json
```

Reply:
@SalDaniele  `./print_results.py ft-logs/*.json` works on the Jenkins workspace, which gets wiped every time.

however, it does not work when you run `make traffic-flow-tests` repeatedly on your machine (without wiping the directory before run). Then result files pile up in `kubernetes-traffic-flow-tests/ft-logs/`. Instead, we only want to compare the last run.

That was the point of `$OUTPUT_BASE` above.

Reply:
let me open a PR